tldw_Server_API/tests/Evaluations/unit/test_connection_pool_shutdown.py

```
@@ -111,17 +131,53 @@ def test_shutdown_warns_if_maintenance_thread_is_still_alive_after_join(
+        monkeypatch: pytest.MonkeyPatch,
+        temp_db_path,
+    ):
+        pool = ConnectionPool(db_path=str(temp_db_path), enable_monitoring=False)
+        maintenance_task = _StopsAfterFinalJoinMaintenanceTask()
+        infos: list[tuple[object, ...]] = []
+        warnings: list[tuple[object, ...]] = []
+        pool._maintenance_task = maintenance_task
+
+        monkeypatch.setattr(
+            "tldw_Server_API.app.core.Evaluations.connection_pool.logger.info",
+            lambda *args, **kwargs: infos.append(args),
```

Ruff (0.15.10)

[warning] Unused lambda argument: `kwargs`

(ARG005)